app/models/concerns/metric_types.rb

```diff
@@ -18,6 +18,10 @@ module MetricTypes
     ],
     'subject.first-to-classify' => [
       'classification_created'
+     ],
+    'classifier.first-to-classify.visible' => [
```

Comment:
OOI is this the first person to ever classify on the project / workflow OR the first classification from each user. If the latter maybe reword the name, `user.first-classification-event` or something.